Managers/UInputMan.cpp

```diff
@@ -981,4 +1012,74 @@ namespace RTE {
 			}
 		}
 	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifdef __unix__
+	void UInputMan::HandleAllegroMouseInput() {
+		if (_xwin.display == 0)
+			return;
+
+		int nEvents = XPending(_xwin.display);
+		std::vector<XEvent> events;
+		events.reserve(nEvents + 10);
+		for (int i = 0; i < XPending(_xwin.display); ++i) {
+			XEvent e;
+			XNextEvent(_xwin.display, &e);
+			events.push_back(e);
+		}
+
+		static int mousePrevX = 0;
```

Comment:
Why are these static? UInputMan is a singleton so there can only ever be one of them 😕

Reply:
I don't entirely understand what you mean by this